server/template.js

```diff
@@ -1,37 +1,84 @@
 import marinate from 'marinate';
 import { ONLINE_COLOR } from '../common/constants';
 
+const preloadImages = [
```

Comment:
Right now these are hard coded but eventually we would be reading these from a manifest file generated by webpack.

Reply:
Do we already have a issue created for the follow-up item we need to do?